tomcat/osmap.yaml

```diff
@@ -49,3 +49,31 @@ Arch:
   catalina_pid: /var/run/tomcat8.pid
   catalina_base: /usr/share/tomcat8
   catalina_home: /usr/share/tomcat8
+  #Not used on Arch
+  manager_pkg:
+MacOS:
+  {% set darwin_javahome = salt['cmd.run']('/usr/libexec/java_home') if grains.os =='MacOS' else {} %}
+  service: homebrew.mxcl.tomcat
```

Comment:
This seems to infer Tomcat always being installed via homebrew, which is not always the case. On the other hand, it doesn't conflict with anything that currently exists since the osmaan wasn't even there to begin with, so it's not an actual current problem. If someone or anyone in the future wants Darwin/macOS with non-brew installed versions they could simply PR or Issue for that.

Reply:
I was not sure best way to handle Java_home lookup.  Great point regarding Tomcat having different software vendors (homebrew is just one).  Needs followup RFE issue. thanks.